### DISCUSSION TITLE
Inheriting layout but adding TOC doesn't show TOC

### DISCUSSION THREAD

**Discussion by TheNathanSpace** [mmistakes/minimal-mistakes#3869] [Q&A]
This layout, when placed in the `_layouts` directory, doesn't create the table of contents.

```yaml
---
layout: single
toc: true
---

{{ content }}
```

However, this works fine on individual pages:
```yaml
---
layout: single
toc: true
---
```

Does anybody know why I can't create a layout inheriting from the `single` layout but with a TOC?

**Comment by iBug** [comment]
I don't think Jekyll reads page variables from their layouts. You might as well use the `defaults` key in `_config.yml`.
